crates/holochain/src/conductor/conductor.rs

```diff
@@ -4214,70 +4256,6 @@ fn query_dht_ops_from_statement(
     Ok(r)
 }
 
-#[cfg_attr(feature = "instrument", tracing::instrument(skip(p2p_evt, handle)))]
```

Comment:
Will this be rewritten?

Reply:
You didn't give me enough context in the diff, i had to go search through the whole diff to find your comment to know what you were asking about : )

No, this will not be rewritten. It is superseded by the conductor struct now implementing HcP2pHandler.

Reply:
Sorry, alright.